setup.py

```diff
@@ -84,21 +84,22 @@ def merge_req_lists(*req_lists: List[str]) -> List[str]:
 ]
 
 otl_dependencies = [
-    "opentelemetry-api==1.17.0",
-    "opentelemetry-exporter-otlp==1.17.0",
-    "opentelemetry-exporter-otlp-proto-grpc==1.17.0",
-    "opentelemetry-exporter-otlp-proto-http==1.17.0",
-    "opentelemetry-instrumentation==0.38b0",
-    "opentelemetry-proto==1.17.0",
-    "opentelemetry-sdk==1.17.0",
-    "opentelemetry-semantic-conventions==0.38b0",
+    "opentelemetry-api>=1.20.0",
+    "opentelemetry-exporter-otlp>=1.20.0",
+    "opentelemetry-exporter-otlp-proto-grpc>=1.20.0",
+    "opentelemetry-exporter-otlp-proto-http>=1.20.0",
+    "opentelemetry-instrumentation>=0.38b0",
+    "opentelemetry-proto>=1.20.0",
+    "opentelemetry-sdk>=1.20.0",
+    "opentelemetry-semantic-conventions>=0.41b0",
```

Comment:
I think it'd be a good practice to add comments as to why there are restrictions.

Reply:
Most of the restrictions will be due to "latest api required".
This needs more context (like "required to export dictionaries").

Reply:
Updated